src/uct/api/uct.h

```diff
@@ -776,22 +777,25 @@ ucs_status_t uct_iface_mem_alloc(uct_iface_h iface, size_t length,
 
 void uct_iface_mem_free(const uct_allocated_memory_t *mem);
 
-/* @ingroup UCT_AM
- * @brief  List of capabilities of active message callback
+/**
+ * @ingroup UCT_AM
+ * @brief AM callback capabilities
+ *
+ * List of capabilities of active message callback
  *
  * A callback must have either SYNC or ASYNC flags.
  */
-enum {
-    UCT_AM_CB_FLAG_SYNC  = UCS_BIT(1), /**< callback is always invoked from the context (thread, process)
+enum uct_am_cb_cap {
```

Comment:
@yosefe Please, look at this. I had to name enum here because it has references from documentation. As I understand we can't make reference to unnamed enum.